### PR TITLE
Relocates gpperfmon reference from GPCC

### PR DESCRIPTION
**Pull request by litzell**
[ci-skip]

### CODE REVIEW

File: gpdb-doc/dita/GPRefGuide.ditamap [hunk 1]
```diff
@@ -289,6 +289,26 @@
 	</chapter>
 	<chapter href="ref_guide/gp_toolkit.xml" navtitle="The gp_toolkit Administrative Schema"
 		id="gp_toolkit"/>
+    <chapter href="ref_guide/gpperfmon/dbref.xml" navtitle="Command Center Database Reference">
+  		<topicref href="ref_guide/gpperfmon/db-database.xml" type="topic"/>
+  		<topicref href="ref_guide/gpperfmon/db-emcconnect-history.xml" type="topic"/>
```
Comment (Chibin) [inline gpdb-doc/dita/GPRefGuide.ditamap]: This probably needs to get removed?

Comment (litzell) [inline gpdb-doc/dita/GPRefGuide.ditamap]: Yes, and the iterators too. Thank you.

**Comment by dyozie** [comment]
Chuck - I realize we don't do this in the docs yet, but shouldn't these new gpcc additions be tagged with otherprops="pivotal"?

**Comment by litzell** [comment]
The gpperfmon_install management utility and agents are open source, so I didn't think to treat this as Pivotal. Perhaps there is a better name than the historical "Command Center Database Reference." It could be "The gpperfmon Database Reference" instead.

**Comment by dyozie** [comment]
Yes - if this could be "rebranded" for gpperfmon I think that would be best.

File: gpdb-doc/dita/ref_guide/gpperfmon/db-overview.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+  PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<topic id="overview">
+   <title>Overview</title>
+   <body>
+      <p>The Command Center database consists of three sets of tables; <codeph>now</codeph> tables
```
Comment (Chibin) [inline gpdb-doc/dita/ref_guide/gpperfmon/db-overview.xml]: There are many mentions of Command Center. Do we plan on removing these mentions or renaming them?

**Comment by litzell** [comment]
Edited instances throughout the docs where the gpperfmon database is called "Command Center database" to clarify that gpperfmon and agents are part of GPDB and GPCC is optional and is a user of gpperfmon.

**Comment by larham** [comment]
Hey Chuck, FYI, we did NOT find any references to "ignore_qexec_packet" in the documentation here.  That option is getting cut in a forthcoming PR.  

We did see a reference to it at old documentation at: https://gpcc.docs.pivotal.io/220/gpcc/topics/configref-agent.html

Can you confirm that there is no reference to the "ignore_qexec_packet" option?

Thanks!

**Comment by litzell** [comment]
There is a topic in the GPCC documentation to document gpperfmon.conf:

http://gpcc.docs.pivotal.io/320/gpcc/topics/configref-agent.html

ignore_qexec_packet is documented in that topic. Also, ignore_qexec_packet was introduced in the GPDB 4.3.4.1 release notes, so it's mentioned in the 4.3. docs, but not in 5.0. 

I think the gpperfmon.conf topic should be moved from GPCC to GPDB along with the database reference docs. I'll move it into the 4.3 docs, including ignore_gexec_packet, then into this repo without that GUC.

**Pull request merged**